egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -596,8 +616,9 @@ while [ $x -lt $num_iters ]; do
         $cmd $train_queue_opt $dir/log/train.$x.$n.log \
           nnet3-train $parallel_train_opts --print-interval=10 --momentum=$momentum \
           --max-param-change=$max_param_change \
-          --optimization.min-deriv-time=$min_deriv_time "$raw" \
-          "ark:nnet3-copy-egs $context_opts ark:$cur_egs_dir/egs.$archive.ark ark:- | nnet3-shuffle-egs --buffer-size=$shuffle_buffer_size --srand=$x ark:- ark:-| nnet3-merge-egs --minibatch-size=$this_num_chunk_per_minibatch --measure-output-frames=false --discard-partial-minibatches=true ark:- ark:- |" \
+          --optimization.min-deriv-time=$min_deriv_time \
+          --num-splits-per-chunk=$[$chunk_width/$minibatch_chunk_size] "$raw" \
```

Comment:
Do you think replacing <code>--num-splits-per-chunk</code> with <code>--num-minibatches-per-chunk</code> would be more understandable ?